modules/vaos/app/services/vaos/configuration.rb

```diff
@@ -14,6 +14,10 @@ def service_name
       'VAOS'
     end
 
+    def cert_key
+      @key ||= OpenSSL::PKey::RSA.new(File.read(Settings.va_mobile.key_path))
+    end
```

Comment:
`VAOS::Configuration` is a singleton, and therefore any instance variables in the singleton are cached for the life of the app... am I following that right? Just want to make sure this doesn't get loaded over and over again.

Secondly, not very important, but I think `key` is a more succinct name for this method.

Reply:
yes it's a singleton, fairly long inheritance chain but the base class is `Common::Client::Configuration::Base`

Reply:
I can confirm, it is a singleton and therefore should not be loaded over and over again.